modules/nw-multus-create-network.adoc

```diff
@@ -24,6 +24,13 @@ network.
 
 .Procedure
 
+. Create the namespace for the additional networks.
```

Comment:
I'd probably note this as optional. Since you could be re-using a namespace that already exists.

I believe the main focus on namespaces for the docs generally specifies that both the target workload pod and the `NetworkAttachmentDefinition` both exist in the same namespace.

Reply:
I would agree that this is optional.

Reply:
@dougbtv: @jboxman-rh told me we should be aware of the differences between creating a namespace and a project. `oc create namespace <name>`  just creates a namespace  
`oc new-project <name>`  actually creates an associated OCP project and some other things in addition to the namespace, and the difference might matter
I discussed this with @clapolt and he agreed creating the namespace would be sufficient. Can you confirm Doug? 
I've changed the step to optional and to `oc create namespace <namespace_name>`

Reply:
I have confirmed that using a namespace works on Power. 

One small thing I'd like to point out is that in the CNO configuration example, the namespace is named "project2" so perhaps that should be changed to "namespace2" for a little more clarity.